edalize/icarus.py

```diff
@@ -90,6 +93,7 @@ def configure_main(self):
                 f.write("VPI_MODULES      := {}\n".format(_vpi_modules))
             f.write("TOPLEVEL         := {}\n".format(self.toplevel))
             f.write("IVERILOG_OPTIONS := {}\n".format(' '.join(self.tool_options.get('iverilog_options', []))))
+            f.write("VPP_OPTIONS := {}\n".format(' '.join(self.tool_options.get('vpp_options', []))))
```

Comment:
In my opinion, this should be
```suggestion
            f.write("VPP_OPTIONS := {}\n".format(' '.join(
                shlex.quote(opt) for opt in self.tool_options.get('vpp_options', [])
            )))
```

Reply:
`IVERILOG_OPTIONS` has the same issue.

Reply:
I can see the point here and I'm happy to take advice from the maintainers. I am a little reluctant to make the change as part of this PR, where I'm trying to keep mods to existing backends to a minimum. It could be opening a can of worms... See for example the GHDL backend:

https://github.com/olofk/edalize/blob/f72286e1c28a7cb23d8228521370b05e94287455/edalize/ghdl.py#L111

and several incidences in the ModelSim backend:

https://github.com/olofk/edalize/blob/f72286e1c28a7cb23d8228521370b05e94287455/edalize/modelsim.py#L157

Reply:
Agreed that this looks like a systemic issue in edalize, and so may be out of scope for this specific path.